lib/taskana-core/src/test/java/acceptance/workbasket/DeleteWorkbasketAccTest.java

```diff
@@ -177,16 +187,45 @@ public void testCascadingDeleteOfAccessItems()
         workbasketService.createWorkbasketAccessItem(accessItem);
         List<WorkbasketAccessItem> accessItemsBefore = workbasketService.getWorkbasketAccessItems(wbId);
         assertEquals(5, accessItemsBefore.size());
-        workbasketService.deleteWorkbasket(wbId);
         try {
+            workbasketService.deleteWorkbasket(wbId);
             workbasketService.getWorkbasket("WBI:100000000000000000000000000000000008");
             fail("There should be no result for a deleted Workbasket.");
-        } catch (WorkbasketNotFoundException e) {
+        } catch (WorkbasketNotFoundException | WorkbasketInUseException e) {
             // Workbasket is deleted
         }
 
         List<WorkbasketAccessItem> accessItemsAfter = workbasketService.getWorkbasketAccessItems(wbId);
         assertEquals(0, accessItemsAfter.size());
     }
 
+    @WithAccessId(userName = "admin", groupNames = {"businessadmin"})
+    @Test
+    public void testMarkWorkbasketForDeletion()
```

Comment:
This can be split up in two tests. This way you can be more precise in the naming of the tests and the expected results. The first one would expext an WorkbasketInUseException, wouldn't it?

Reply:
I don't split this test on two test because already exist a method that test if the workbasket can be removed